pkg/registry/batch/job/storage/storage.go

```diff
@@ -117,3 +118,13 @@ func (r *StatusREST) Update(ctx context.Context, name string, objInfo rest.Updat
 func (r *StatusREST) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set {
 	return r.store.GetResetFields()
 }
+
+func (r *StatusREST) Delete(ctx context.Context, name string, deleteValidation rest.ValidateObjectFunc, options *metav1.DeleteOptions) (runtime.Object, bool, error) {
```

Comment:
should be REST, not StatusREST

Reply:
this will not catch calls to DeleteCollection that omit propagation options

Reply:
Oops. I thought I changed it to REST. Thank you for catching it.